src/components/molecules/SfGallery/README.md

```diff
@@ -0,0 +1,52 @@
+# SfComponent
+
+<!-- Write about general purpose of the component. Include screenshot (to be replaced with a live example once we migrate to vuepress) -->
```

Comment:
can you move this to summary section of info storybook plugin? (see examples from other stuff like filters)